app/services/phone_setup_recaptcha_validator.rb

```diff
@@ -0,0 +1,30 @@
+class PhoneSetupRecaptchaValidator < RecaptchaValidator
```

Comment:
It looks like we only ever instantiate the subclass but never the parent clas?

I feel like with one one concrete use case, this is not a great place for a subclass/superclass relation? Are we planning to have a bunch of these? If we do want to two separate classes, what if we delegated to a RecaptchaValidator instance instead of subclassing?

Reply:
Aw, and here I was all proud of my abstraction 😂  Yes, this is currently the only use-case. Also yes, it was me prematurely anticipating that there may be some future reuse. But I think it also creates a clean separation of concerns/responsibility?

For your final point, are you suggesting something like ... ?

```rb
class PhoneSetupRecaptchaValidator
  attr_reader :validator

  def initialize(...)
    @validator = RecaptchaValidator.new
  end

  delegate :valid?, to: :validator
end
```

Reply:
Updated to delegate in 1868839de. In some ways I like this, especially in how it makes the `score_threshold` more explicitly assigned in the constructing of the delegate instance. But I also don't like it in that it seems like imitating inheritance for the sake of avoiding a subclass relation 😕